lib/drivelist.js

```diff
@@ -46,14 +47,14 @@ const scripts = require('./scripts');
  */
 exports.list = (callback) => {
   const operatingSystem = os.platform();
-  const script = scripts.paths[operatingSystem];
+  const script = scripts[operatingSystem];
```

Comment:
Oh right, even more reason to give it a variable name that better reflects what it is, and that it's not just a filename the way that it used to be ;)